topics:
  state_estimation: "/coat_mpc/state_estimation"
  lap_counter: "/coat_mpc/lap_counter"
  penalty_and_laps: "/coat_mpc/penalty_and_laps"
  track: "/track_info"
  namespace: "track_center"

simulation:
  package: "mpc_launch"
  launch: "simulation.launch"

run:
  package: "mpc_launch"
  launch: "run.launch"

frames:
  world: "world"
  base_link: "vehicle"
